spec/samples/creatives.js

```diff
@@ -46,6 +49,11 @@ export const creatives = `<Creatives>
             <IconClickThrough><![CDATA[http://example.com/linear-clickthrough]]></IconClickThrough>
             <IconClickTracking id='icon-click-1'><![CDATA[http://example.com/linear-clicktracking1]]></IconClickTracking>
             <IconClickTracking id='icon-click-2'><![CDATA[http://example.com/linear-clicktracking2]]></IconClickTracking>
+            <IconClickFallbackImages>
+              <IconClickFallbackImage width="10px" height="10px">
+                <![CDATA[http://adserver.com/fallback.png]]>
+              </IconClickFallbackImage>
+            </IconClickFallbackImages> 
```

Comment:
We may need a use case with multiple <IconClickFallbackImage> to fully cover use cases